Documentation/getting-started-ubuntu.md

```diff
@@ -5,7 +5,7 @@ The following guide will show you how to build and run the sample [etcd ACI](htt
 ## Download and start an Ubuntu Wily box
```

Comment:
Good catch, there's a couple of other Wily pointers in it too.

Nevertheless we (@lucab and me) had the idea to delete this file completely. It is redundant with the `trying-out-rkt` instructions, and once rkt is fixed in Debian we could document the installation there?